src/utils/uiConfig.js

```diff
@@ -64,15 +64,16 @@ export function useChainsConfig(): { [key: Chain]: ChainConfig } {
       iconName: 'binance',
       color: colors.binance,
       gasSymbol: BNB,
-      iconUrl: 'https://images.prismic.io/pillar-app/d8e73973-82f1-4b20-a068-fe20e21816b0_0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c_1.png?auto=compress,format',
+      iconUrl:
+        'https://images.prismic.io/pillar-app/d8e73973-82f1-4b20-a068-fe20e21816b0_0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c_1.png?auto=compress,format',
     },
     [CHAIN.XDAI]: {
       title: t('chains.xdai'),
       titleShort: t('chainsShort.xdai'),
       iconName: 'xdai',
       color: colors.xdai,
       gasSymbol: XDAI,
-      iconUrl: 'https://tokens.1inch.exchange/0x6b175474e89094c44da98b954eedeac495271d0f.png',
+      iconUrl: 'https://tokens.1inch.exchange/0x6810e776880c02933d47db1b9fc05908e5386b96.png',
```

Comment:
As above: Here's the icon kit for xDai: https://developers.gnosischain.com/media/media-kit#xdai-stable-coin

Reply:
I set the Gnosis icon as per zeplin ui.
![image](https://user-images.githubusercontent.com/78720269/186863442-81fff798-8d79-45f8-980a-01477aefa300.png)

Reply:
Whoops sorry i thought this was the native token